### ISSUE TITLE
soundmon files not working

### ISSUE THREAD

**Issue by ciros88** [aargirakis/BZRPlayer#519]
seems that bzr2 try to play them with FLOD instead of UADE
[#soundmon.zip](https://github.com/aargirakis/BZRPlayer/files/15105191/soundmon.zip)

**Comment by aargirakis** [comment]
Soundmon 1.0 files with missing samples

**Comment by ciros88** [comment]
you are right, only JARRE.SYNTH works (just some silence at the beginning). no issues at all here
